src/main/kotlin/racingcar/InputView.kt

```diff
@@ -0,0 +1,19 @@
+package racingcar
+
+import camp.nextstep.edu.missionutils.Console
+
+class InputView {
+    // 자동차 이름 입력 받기
+    fun getCarNames(): List<String> {
+        println("경주할 자동차 이름을 입력하세요. (이름은 쉼표(,) 기준으로 구분)")
+        val input = Console.readLine() ?: throw IllegalArgumentException("자동차 이름 입력이 필요합니다.")
+        return input.split(",").map { it.trim() }
```

Comment:
변수를 선언 과정을 한번 거친 후 리턴해주는 것도 하나의 방법이지 않을까?🤔 생각해봅니다!
```suggestion
        val carNames = input.split(",").map { it.trim() }
        return carNames
```

Reply:
좋은 지적인 거 같습니다! 더 의미를 잘 알수 있네요